google/ggrc-core
# GGRC-7911 Remove the line under the last object's title in the object list in modal popups

# Issue description

The issue connected to result's list on mapper-results (reproduced on verify modal, mapping modal, global search). A line displays below the last object's title in the list, however, it doesn't display under others' above in the expanded mode. 

# Steps to test the changes

1. Open GGRC
2. Click Global Search button
3. Set any Search query, e.g. Object Type as "Control"
4. Click 'Search' button
5. In the Search results list expand the first object and the last one
6. Check the underlining below each object's title 
7. Expected: Last object's underline displays under expanded info.

# Solution description

Changed styles for mapper-results component.

# Sanity checklist




# PR Review checklist
